### PR TITLE
Update feature page: List VMs pinned to a host

### PR DESCRIPTION
**Pull request by akrejcir**
Changes proposed in this pull request:
- Update the 'List VMs pinned to a host' feature page to reflect current implementation state.

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @akrejcir

This pull request needs review by: @pcbailey @vojtechszocs

### CODE REVIEW

File: source/develop/release-management/features/sla/list-vms-pinned-to-host.html.md [hunk 1]
```diff
@@ -41,12 +41,20 @@ Possibilities to expose the new query through the REST API:
   ```
 
 ### Webadmin
-There will be two tables in the `Virtual Machines` sub-tab in the host details page.
-One for running VMs and the other for pinned VMs:
+The table in the `Virtual Machines` sub-tab in the host details page shows
```
Comment (pcbailey) [inline source/develop/release-management/features/sla/list-vms-pinned-to-host.html.md]: Small suggestion here: "in the host details..." -> "on the host details..."

File: source/develop/release-management/features/sla/list-vms-pinned-to-host.html.md [hunk 2]
```diff
@@ -41,12 +41,20 @@ Possibilities to expose the new query through the REST API:
...
+running VMs and pinned VMs. A radio button on top of it can be used to
+show only one of the groups.
 
-![](/images/wiki/VMs_pinned_to_host.png)
```
Comment (pcbailey) [inline source/develop/release-management/features/sla/list-vms-pinned-to-host.html.md]: I think you need to add a 3px top margin to the label to fix the vertical alignment. If you look at the radio button group label in the Disks sub tab of the VMs main tab, you'll see the difference.

Comment (akrejcir) [reply source/develop/release-management/features/sla/list-vms-pinned-to-host.html.md]: Good point. Now it looks better.

Comment (pcbailey) [reply source/develop/release-management/features/sla/list-vms-pinned-to-host.html.md]: Agreed! =) You get a +1 from me.

**Comment by vojtechszocs** [comment]
> Overall, I really like the look of this approach much better than the two tables approach.

+1 it's cleaner to have one table, reflecting a specific criteria, than to have multiple tables. This is based on the assumption that users need to see particular data one-criteria-at-a-time, instead of seeing everything :smile:

File: source/develop/release-management/features/sla/list-vms-pinned-to-host.html.md [hunk 3]
```diff
@@ -41,12 +41,20 @@ Possibilities to expose the new query through the REST API:
   ```
 
 ### Webadmin
-There will be two tables in the `Virtual Machines` sub-tab in the host details page.
-One for running VMs and the other for pinned VMs:
+The table in the `Virtual Machines` sub-tab on the host details page shows
```
Comment (vojtechszocs) [inline source/develop/release-management/features/sla/list-vms-pinned-to-host.html.md]: In oVirt 4.2 UI we moved away from main & sub tab terminology (even though the code still has that naming convention).

Please refer to tabs under details of a specific entity as "detail tabs", e.g. VM detail tab of host entity.

Comment (akrejcir) [reply source/develop/release-management/features/sla/list-vms-pinned-to-host.html.md]: Done

**Comment by gregsheremeta** [comment]
ancient, closing. feel free to update and re-push

**Pull request closed without merging**